### PR TITLE
@W-8841422@: RetireJS now determines file eligibility by checking con…

### PR DESCRIPTION
**Pull request by jfeingold35**
…tents instead of metadata MIME type.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,12 +20,14 @@
 		"find-java-home": "^1.1.0",
 		"globby": "^11.0.0",
 		"html-escaper": "^3.0.0",
+		"is-zip": "^1.0.0",
+		"istextorbinary": "^5.12.0",
```
Comment (jfeingold35) [inline package.json:24]: I'm going to create a 3PP for these libraries shortly.

Comment (jfeingold35) [reply package.json:24]: istextorbinary has been approved: https://gus.lightning.force.com/lightning/r/ADM_Third_Party_Software__c/a0qB0000000VI8aIAG/view
Still waiting on is-zip.

Comment (jfeingold35) [reply package.json:24]: is-zip is approved: https://gus.lightning.force.com/lightning/r/ADM_Third_Party_Software__c/a0qB0000000VI8fIAG/view

**Review by jbartolotta-sfdc — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -20,12 +20,14 @@
...
 		"mustache": "^4.0.1",
 		"node-stream-zip": "1.11.3",
 		"normalize-path": "^3.0.0",
 		"picomatch": "^2.2.2",
 		"reflect-metadata": "^0.1.13",
-		"retire": "^2.2.3",
+		"retire": "^2.2.4",
```
Comment (ranekere-sfdc) [inline package.json:30]: Do we need to update the versions.yml for this retire upgrade?

Comment (jfeingold35) [reply package.json:30]: Good catch. I'll get a PR ready shortly.

**Pull request merged**